sympy/stats/crv_types.py

```diff
@@ -2058,8 +2114,11 @@ def Nakagami(name, mu, omega):
 class NormalDistribution(SingleContinuousDistribution):
     _argnames = ('mean', 'std')
 
+    set = Interval.open(-oo, oo)
+
     @staticmethod
     def check(mean, std):
+        _value_check(mean.is_real, "Location or mean should be real.")
```

Comment:
~Location or~ If someone is calling this with 'Location' perhaps they could have their own `check`?